integration/tests/cook/test_basic.py

```diff
@@ -96,8 +96,11 @@ def test_failing_submit(self):
 
     def test_progress_update_submit(self):
         job_executor_type = util.get_job_executor_type(self.cook_url)
-        command = 'echo "progress: 25 Twenty-five percent"; sleep 1; exit 0'
-        job_uuid, resp = util.submit_job(self.cook_url, command=command, executor=job_executor_type, max_runtime=60000)
+        command = 'echo "progress: 25 Twenty-five percent in ${CUSTOM_PROGRESS_OUTPUT_FILE}"; sleep 1; exit 0'
+        job_uuid, resp = util.submit_job(self.cook_url, command=command,
+                                         env={'EXECUTOR_PROGRESS_OUTPUT_FILE_ENV': 'CUSTOM_PROGRESS_OUTPUT_FILE',
```

Comment:
Should we just add these to the default job env?

Reply:
I'd rather the test configure it explicitly.

Reply:
Sounds good, just wanted to check